### PR TITLE
Update ImageCapture web fundamentals entry 

### PR DESCRIPTION
**Pull request by yellowdoge**
Originally published [here](https://developers.google.com/web/updates/2016/12/imagecapture), needs an update because is being shipped in M59 and some things have changed.

Needs some polishing of the literature, but I'm uploading it anyhow so that (any of) you guys can take a look.

PTAL:
+@beaufortfrancois (all your comments from the early review taken in except the `revokeURL()`, thanks!)
+@samdutton

FYI:
+@petele
+@owencm

### CODE REVIEW

**Comment by beaufortfrancois** [comment]
LGTM with nit: The timeout is still required at that time, we should document it and figure out what to do with https://github.com/GoogleChrome/samples/pull/503.

File: src/data/_contributors.yaml [hunk 1]
```diff
@@ -1329,3 +1329,19 @@ maciejzasada:
     - author
   google: "104812516714503133849"
   email: maciej@unit9.com
+
+mcasas:
```
Comment (petele) [inline src/data/_contributors.yaml]: `_contributors.yaml` needs to be sorted by family name, would you please adjust.

File: src/content/en/updates/2016/12/imagecapture.md [hunk 2]
```diff
@@ -86,22 +84,102 @@ it on a `<canvas>`:
...
+      zoomSlider.min = capabilities.zoom.min;
+      zoomSlider.max = capabilities.zoom.max;
+      zoomSlider.step = capabilities.zoom.step;
+      zoomSlider.value = settings.zoom;
+    }
+
+Note: `MediaStreamTrack.getCapabilities()` and `.getSettings()` are synchronous
+but the capabilities are only available when the camera starts actually
+streaming, so make sure there is a delay between calling these methods and
+`getUserMedia()`, see [crbug.com/711524](https://crbug.com/711524).
```
Comment (yellowdoge) [inline src/content/en/updates/2016/12/imagecapture.md]: @beaufortfrancois this note will stand out in the article (as it does e.g. in https://developers.google.com/web/updates/2016/12/imagecapture); I'm not sure how to make the necessary timeout between `getUserMedia()` and `getCapabilities()/getSettings()` more evident than this in the current snippets.  I will add the timeout and a link to the bug in https://github.com/GoogleChrome/samples/pull/503.

File: src/data/_contributors.yaml (hunk 1 above)
Comment (yellowdoge) [inline src/data/_contributors.yaml]: Done!

File: src/content/en/updates/2016/12/imagecapture.md (hunk 2 above)
Comment (beaufortfrancois) [inline src/content/en/updates/2016/12/imagecapture.md]: Perfect then

**Pull request merged**